src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,56 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int [] values = readFromfile(fromFileName);
```

Comment:
```suggestion
        int[] values = readFromfile(fromFileName);
```
use IDE auto formatter before committing